src/abstractdataframe/abstractdataframe.jl

```diff
@@ -956,20 +956,38 @@ Base.hcat(df1::AbstractDataFrame, df2::AbstractDataFrame, dfn::AbstractDataFrame
     hcat!(hcat(df1, df2, makeunique=makeunique), dfn..., makeunique=makeunique)
 
 """
-    vcat(dfs::AbstractDataFrame...)
+    vcat(dfs::AbstractDataFrame...; columns::Union{Symbol, AbstractVector{Symbol}}=:equal)
 
-Vertically concatenate `AbstractDataFrames`.
+Vertically concatenate `AbstractDataFrame`s.
 
-Column names in all passed data frames must be the same, but they can have
-different order. In such cases the order of names in the first passed
-`DataFrame` is used.
+The `columns` keyword argument determines the columns of the returned data frame:
+
+* `:equal` (the default): require all data frames to have the same column names.
```

Comment:
We had a discussion with @nalimilan about the best name for this option. Alternatives are `:identical` or `:same`. @pdeffebach - do you have any opinion? (all are OK, but it would be good to choose the most intuitive)

Reply:
let's go with `:same`